zanata-war/src/main/webapp/WEB-INF/layout/dashboard/projects.xhtml

```diff
@@ -26,15 +26,17 @@
               <span class="is-invisible">#{msgs['jsf.dashboard.projects.maintainedProjects.label']}</span>
             </a>
           </li>
-          <li class="tab__end">
-            <a href="#" jsfc="s:link"
-              id="create-project-link"
-              view="/project/create_project.xhtml"
-              title="#{msgs['jsf.dashboard.projects.createNewProject.label']}">
-              <i aria-hidden="true" class="i i--plus i i--large"></i>
-              <span class="is-invisible">#{msgs['jsf.dashboard.projects.newProject.label']}</span>
-            </a>
-          </li>
+          <s:fragment rendered="#{s:hasPermission(projectHome.instance, 'insert')}">
```

Comment:
Please use identity.hasPermission instead of s:hasPermissin. In the CDI migration all such seam specific EL needs to be migrated